schema.sql

```diff
@@ -10,4 +10,33 @@
 
 -- for the second day's project
 
```

Comment:
- [ ] Please you should create query to answer the following questions :- 
- * What animals belong to Melody Pond?
- * List of all animals that are pokemon (their type is Pokemon). 
- * List all owners and their animals, remember to include those that don't own any animal. How many animals are there per species?
- List all Digimon owned by Jennifer Orwell.
- List all animals owned by Dean Winchester that haven't tried to escape.
- Who owns the most animals? 
- please use join to write these queries

Reply:
Hi @MeqdamAlqudah , Can you please specify the lines of codes that has problems?

Reply:
Hey @IsmailChangezi I hope you are good :+1:
Actually, he cannot specify the lines because you haven't created those queries which he has written above :arrow_up: Kindly make those queries using the SELECT statement :+1: to follow the project requirements :blush: